hypershift-operator/controllers/nodepool/manifests.go

```diff
@@ -60,11 +60,20 @@ func TokenSecret(namespace, name, payloadInputHash string) *corev1.Secret {
 	}
 }
 
-func TuningConfigMap(namespace, name string) *corev1.ConfigMap {
```

Comment:
it is not needed but I thought it would be clearer to name it after the TuneD instead of using the "tuning" which could also refer to "performance profile"

Reply:
makes sense